clang/test/Driver/pch-include-sycl.cpp

```diff
@@ -0,0 +1,33 @@
+// This test checks that a PCH(Precompiled Header) file is 
+// included while performing host compilation in -fsycl mode.
+// The PCH file is created without the -fsycl option.
+
+// RUN: touch %t.h
+
+// RUN: %clang -c -x c++-header %t.h
```

Comment:
Why run the compiler?  Why not just `touch %t.h.gch`?  It minimizes the dependency of the compiler running to produce an output file and keeps the behavior solely in the driver functionality.